ktransw.cmd

```diff
@@ -1,17 +1,33 @@
 @echo off
+
 REM
+
 REM Copyright (c) 2016, G.A. vd. Hoorn
+
 REM
+
 REM Licensed under the Apache License, Version 2.0 (the "License");
+
 REM you may not use this file except in compliance with the License.
+
 REM You may obtain a copy of the License at
+
 REM
+
 REM     http://www.apache.org/licenses/LICENSE-2.0
+
 REM
+
 REM Unless required by applicable law or agreed to in writing, software
+
 REM distributed under the License is distributed on an "AS IS" BASIS,
+
 REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
 REM See the License for the specific language governing permissions and
+
 REM limitations under the License.
+
 REM
```

Comment:
This seems to introduce a lot of empty lines. Is this a line-ending conflict?

Reply:
Rossum.cmd was fine in my editor. ktransw.cmd had no carriage returns for me, just one line.

Reply:
>  ktransw.cmd had no carriage returns for me, just one line

You may have a global setting for line-endings that is (trying to) overriding what the repository is configured for. See whether [Dealing with line endings](https://help.github.com/en/articles/dealing-with-line-endings) sheds some light on this.